README.md

```diff
@@ -139,22 +140,24 @@ Spin up a development server with multiple features like `hot-module-replacement
 ```
 $ preact watch
 
-    --src           Specify source directory  (default src)
-    --cwd           A directory to use instead of $PWD  (default .)
-    --esm           Builds ES-2015 bundles for your code.  (default true)
-    --sw            Generate and attach a Service Worker  (default false)
-    --json          Generate build stats for bundle analysis
-    --https         Run server with HTTPS protocol
-    --key           Path to PEM key for custom SSL certificate
-    --cert          Path to custom SSL certificate
-    --cacert        Path to optional CA certificate override
-    --prerender     Pre-render static content on first run
-    --template      Path to custom HTML template
-    --refresh       Will use [`Preact-refresh`](https://github.com/JoviDeCroock/preact-refresh) to do hot-reloading
-    -c, --config    Path to custom CLI config  (default preact.config.js)
-    -H, --host      Set server hostname  (default 0.0.0.0)
-    -p, --port      Set server port  (default 8080)
-    -h, --help      Displays this message
+    --src              Specify source directory  (default src)
```

Comment:
I added more spacing before the flag descriptions, firstly, to match the spacing found under `build`, and secondly the `prerenderUrls` flag was scraping it once added. Extra spacing helped with the readability.